internal/controller/platform/kserve/nimservice.go

```diff
@@ -577,7 +583,7 @@ func (r *NIMServiceReconciler) addGPUResources(ctx context.Context, nimService *
 	// if deployed as multi-node, use the GPU per worker value to assign GPU resources to each worker
 	// TODO auto determine base on tp*pp/(.spec.multiNode.size)
 	if nimService.Spec.MultiNode != nil {
```

Comment:
i think this entire multi-node check should be removed. This function is for auto assignment of GPUs, which holds true for multi-node as well when a trtllm profile is selected. We don't support DRA for auto-assignment anyway.